client/asset/btc/spv.go

```diff
@@ -58,18 +58,20 @@ var _ BTCWallet = (*btcSPVWallet)(nil)
 
 // createSPVWallet creates a new SPV wallet.
 func createSPVWallet(privPass []byte, seed []byte, bday time.Time, dataDir string, log dex.Logger, extIdx, intIdx uint32, net *chaincfg.Params) error {
-	dir := filepath.Join(dataDir, net.Name, "spv")
-	if err := logNeutrino(dir); err != nil {
+	netDir := filepath.Join(dataDir, net.Name)
+	walletDir := filepath.Join(netDir, "spv")
+
+	if err := logNeutrino(netDir); err != nil {
 		return fmt.Errorf("error initializing btcwallet+neutrino logging: %w", err)
 	}
 
-	logDir := filepath.Join(dir, logDirName)
+	logDir := filepath.Join(netDir, logDirName)
 	err := os.MkdirAll(logDir, 0744)
```

Comment:
Not something with this PR per se, but what are these two `Join`/`MkdirAll` lines doing here after `logNeutrino` anyway?

Reply:
Ah, yeah. I think I wanted to remove this. Because `logNrutrino` will create the directory.